.changeset/selfish-dancers-sort.md

```diff
@@ -0,0 +1,6 @@
+---
+"vercel": patch
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** I think we should call this a minor because it changes some behavior, but not technically in a backwards incompatible way (because you have to migrate your user account to encounter the new guards).‏